CMakeTests/AddCompileDefinitions.cmake

```diff
@@ -0,0 +1,42 @@
+# Add a C or C++ compile definitions to the current scope
+#
+# add_compile_definitions(def [def ...] [DEBUG | RELEASE])
```

Comment:
DEBUG and RELEASE just can't be the last argument then. It's the tricky part of argument parsing in CMake, I couldn't find a good solution that was terse enough.